src/main/scala/mesosphere/marathon/core/flow/impl/ReviveOffersActor.scala

```diff
@@ -3,15 +3,16 @@ package core.flow.impl
 
 import java.time.Clock
 
-import akka.actor.{ Actor, Cancellable, Props }
-import akka.event.{ EventStream, LoggingReceive }
+import akka.actor.{Actor, Cancellable, Props}
+import akka.event.{EventStream, LoggingReceive}
 import com.typesafe.scalalogging.StrictLogging
 import mesosphere.marathon.MarathonSchedulerDriverHolder
 import mesosphere.marathon.core.flow.ReviveOffersConfig
 import mesosphere.marathon.core.flow.impl.ReviveOffersActor.OffersWanted
-import mesosphere.marathon.core.event.{ SchedulerRegisteredEvent, SchedulerReregisteredEvent }
+import mesosphere.marathon.core.event.{SchedulerRegisteredEvent, SchedulerReregisteredEvent}
+import mesosphere.marathon.metrics.{Metrics, ServiceMetric}
 import mesosphere.marathon.state.Timestamp
-import rx.lang.scala.{ Observable, Subscription }
+import rx.lang.scala.{Observable, Subscription}
```

Comment:
The imports should be re-formatted.

Reply:
What Ivan said ^ . The format should not have been changed and causes problems with code updates during builds.

Reply:
Oh, this was IntelliJ auto-formatting. I guess I have to configure